### ISSUE TITLE
Support for test-only runs for Snapshot using test binary only

### ISSUE THREAD

**Issue by BJap** [fastlane/fastlane#12723]
#### Motivation Behind Feature
For larger apps, it makes sense to split localization testing into multiple CI projects. Xcode 9 allows for a build to create a test binary at a provided derived data path, and upload that artifact for use in other jobs. It is undesirable to build for every single one of them, and, the version control system is not build to handle such a load. Essentially build once, test everywhere.

#### Feature Description
Currently it appears this is not possible. The feature exists to build for testing, and then test without building in the same folder space, however, the full use of this feature does not appear to be possible as Apple has provided. 

One can zip up the derived data, and upload for testing. Then, one can download that artifact and run Snapshot. This is as far as it goes. The first issue is that the `xcodeproj` file is required for interaction and so the test fails immediately. If this is provided by also including the file in the zip, the test will still fail as shown.

```
{:PLATFORM_NAME=>nil, :LANE_NAME=>"lanename"}
'latest_simulator_version_for_device': [!] undefined method 'os_version' for nil:NilClass (NoMethodError)
```

#### Alternatives or Workarounds
Currently I do not know any workarounds but one may exist. I am unsure if there are other files I can include to handle `PLATFORM_NAME` or make this run.

It is undesirable to zip up the entire repository. There's an extra 3GB that would be uploaded for a total of about 5GB every time a new build is made, and also downloaded to each test machine from the host artifact host.

It is also undesirable to build and test in every CI job. Some of these will want to rerun on the same build or run on demand and building first and only once saves lots of time.

**Comment by BJap** [comment]
Thus far I have been able to work around this only by pulling the repo in the test job, and downloading the derived data from the cloud storage point. This doesn't save bandwidth so every job puts strain on the git source server, but it does save time not having to build in every test job.

**Comment by janpio** [comment]
Did I understand correctly that you are using `test_without_building` (See https://docs.fastlane.tools/actions/capture_ios_screenshots/#parameters) which is doing its intended job of not building again, but you have additional requirements? What exactly is going wrong?

**Comment by mervecan** [comment]
I have same issue in Android when running `fastlane android aws_device_run_android`

**Comment by janpio** [comment]
And what is that "same issue" @mervecan? The command you posted is a lane, could you please post its content so we have a chance to understand what is going on?

**Comment by mervecan** [comment]
Sorry, my mistake. I'm trying to understand `fastlane` and `aws_device_farm`.  When I run `fastlane android aws_device_run_android`. I get the same error 

```
➡️  Support for test-only runs for Snapshot using test binary only
    https://github.com/fastlane/fastlane/issues/12723 [open] 4 💬
    a day ago

➡️  Problem about "Your app preview can't be uploaded because you have selected to use an app preview from a larger display size."
    https://github.com/fastlane/fastlane/issues/10564 [open] 18 💬
    2 weeks ago

➡️  undefined method `[]' for nil:NilClass during Match
    https://github.com/fastlane/fastlane/issues/10960 [closed] 19 💬
    20 Jan 2018
```

<details>
  <pre><details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                                          |
| --------------------------- | -------------------------------------------------------------- |
| OS                          | 10.13.3                                                        |
| Ruby                        | 2.3.3                                                          |
| Bundler?                    | false                                                          |
| Git                         | git version 2.18.0                                             |
| Installation Source         | /usr/local/bin/fastlane                                        |
| Host                        | Mac OS X 10.13.3 (17D47)                                       |
| Ruby Lib Dir                | /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib |
| OpenSSL Version             | LibreSSL 2.2.7                                                 |
| Is contained                | false                                                          |
| Is homebrew                 | false                                                          |
| Is installed via Fabric.app | false                                                          |
| Xcode Path                  | /Library/Developer/CommandLineTools/                           |
| Xcode Version               |                                                                |


### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   | en_US.UTF-8 | ✅ |
| LANGUAGE |             |   |


### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
# Customise this file, documentation can be found here:
# https://github.com/fastlane/fastlane/tree/master/fastlane/docs
# All available actions: https://docs.fastlane.tools/actions
# can also be listed using the `fastlane actions` command

# Change the syntax highlighting to Ruby
# All lines starting with a # are ignored when running `fastlane`

# If you want to automatically update fastlane if a new version is available:
# update_fastlane

# This is the minimum version number required.
# Update this, if you use features of a newer version
fastlane_version "2.28.3"

default_platform :android

fastlane_require 'json'
fastlane_require 'uri'
fastlane_require 'net/http'

module BuildType
  Store = "store"
  Beta = "beta"
  Other = "other"
end

module Constants
  PROJECT_DIR = "./XYZ/"
end

module Status
  Merged = "Merged"
  Staged = "Staged"
end

module Transition
  ToStaged = "Stage"
  ToVerification = "Testable"
end

platform :android do
  current_branch = nil

  before_all do |lane, options|
    # Init git
    sh "git config user.email \"${GITLAB_USER_EMAIL}\""
    sh "git config user.name \"${GITLAB_USER_LOGIN}\""
    sh "git remote set-url origin https://XYZadmin:${RUNNER_API_TOKEN}@gitlab.XYZ.io/${CI_PROJECT_PATH}.git"
  end

  desc "Runs all unit tests"
  lane :runUnitTests do
    gradle(
      task: "testXYZDebugUnitTestCoverage",
      project_dir: Constants::PROJECT_DIR,
      system_properties: {
        "kotlin.compiler.execution.strategy" => "in-process"
      }
    )
  end

  desc "Run lint checks"
  lane :lintChecks do
    gradle(
      task: "lint",
      flavor: "XYZ",
      build_type: "Alpha",
      project_dir: Constants::PROJECT_DIR,
      system_properties: {
        "kotlin.compiler.execution.strategy" => "in-process"
      }
    )
  end

  desc "Run Sonarqube Analysis. Supply the `previewOnly` option to only scan for preview."
  lane :sonarAnalysis do |options|
    if options[:scannerToken] && options[:commitSha] && options[:commitRefName] && options[:projectId] && options[:userToken]
      scannerToken = options[:scannerToken]
      commitSha = options[:commitSha]
      commitRefName = options[:commitRefName]
      projectId = options[:projectId]
      userToken = options[:userToken]
      
      systemProps = {
          "sonar.login" => scannerToken,
          "sonar.gitlab.project_id" => projectId,
          "sonar.gitlab.user_token" => userToken,
      }
      
      if options[:previewOnly] == "1"
        systemProps["sonar.dynamicAnalysis"] = "resuseReports"
        systemProps["sonar.gitlab.commit_sha"] = commitSha
        systemProps["sonar.gitlab.ref_name"] = commitRefName
        systemProps["sonar.gitlab.json_mode"] = "CODECLIMATE"
        systemProps["sonar.analysis.mode"] = "preview"
        systemProps["sonar.gitlab.failure_notification_mode"] = "commit-status"
        systemProps["sonar.gitlab.only_issue_from_commit_file"] = true
      end

      gradle(
        task: "sonarqube",
        project_dir: Constants::PROJECT_DIR,
        system_properties: systemProps
      )
      next
    end
    raise "Not all properties are passed in, expected 'scannerToken', 'commitSha', 'commitRefName', 'projectId' and 'userToken'"
  end

  desc "Deploy a specific branch to the builds page"
  lane :build do |options|
    if options[:build_type]
        if options[:branch]
            buildType = options[:build_type]
            branch = options[:branch]

            customCompany = "XYZ"
            if options[:company]
              customCompany = options[:company]
            end

            # Build and publish the app
            buildAndPublish(branch: branch, build_type: buildType, company: customCompany)
          next
        end
      raise "No branch specified"
    end
    raise "No build type provided"
  end

  desc "Build the 'other' build type and upload to S3"
  lane :buildOther do |options|
    if options[:branch]
      branch = options[:branch]
      build(build_type: "other", branch: branch)

      next
    end
    raise "No branch found to build"
  end

  desc "Build a beta build and upload to S3. If there is a ticket specified, it will be moved if possible"
  lane :buildBeta do |options|
    if options[:branch]
      branch = options[:branch]
      build(build_type: "beta", branch: branch)

      next
    end
    raise "No branch found to build"
  end

  desc "Build a store build and upload to S3. If there is a ticket specified, it will be moved if possible"
  lane :buildStore do |options|
    if options[:branch]
      branch = options[:branch]
      build(build_type: "store", branch: branch)
      next
    end
    raise "No branch found to build"
  end

  desc "Build a store build for Schneider and upload to S3. No tickets will be moved"
  lane :buildSchneiderStore do |options|
    if options[:branch]
      branch = options[:branch]
      build(build_type: "store", branch: branch, company: "Schneider")
      next
    end
    raise "No branch found to build"
  end

  desc "Build a store build for JivePresent and upload to S3. No tickets will be moved"
  lane :buildJivePresentStore do |options|
    if options[:branch]
      branch = options[:branch]
      build(build_type: "store", branch: branch, company: "Jivepresent")
      next
    end
    raise "No branch found to build"
  end


  desc "Create a build and publish to S3. A branch and build_type are required for this lane. a jira_ticket can be passed as an optional parameter"
  private_lane :buildAndPublish do |options|
    if options[:branch] && options[:build_type] && options[:company]
      company = options[:company]
      branch = options[:branch]
      buildType = options[:build_type]
      jiraIssue = options[:jira_ticket] ? options[:jira_ticket] : ""

      buildNumber = createBuildNumberFromBranch()
      revision = buildNumber
      cleanBranch = cleanBranchName(branch)

      androidBuildType = createAndroidBuildType(buildType)
      companyLowerCase = company.downcase

      if companyLowerCase == "schneider"
        companyLowerCase = "XYZ"
      end

      puts "Project name: #{companyLowerCase}"

      gradle(
        task: "release",
        flavor: company,
        build_type: androidBuildType,
        flags: "--stacktrace",
        project_dir: Constants::PROJECT_DIR,
        properties: {
          "release" => true,
          "revision" => buildNumber,
          "branch" => branch,
          "buildType" => buildType,
          "projectName" => companyLowerCase
        },
        system_properties: {
          "kotlin.compiler.execution.strategy" => "in-process"
        }
      )
    end
  end

  desc "Generate deploy notes for this release. It will update jira and confluence"
  lane :generateReleaseNotes do |options|
    if options[:branch]
      branch = options[:branch]
      version = branch.partition('/').last
      generateReleaseNotes(version)
      next
    end
    raise "No branch specified"
  end

lane :aws_device_run_android do
  ENV['AWS_ACCESS_KEY_ID']     = '----'
  ENV['AWS_SECRET_ACCESS_KEY'] = '----'
  ENV['AWS_REGION']            = 'us-west-2'

  #Build Debug App + Instrumentation Apk
  gradle(
project_dir: Constants::PROJECT_DIR,
task: 'assembleXYZDebug')
  gradle(
project_dir: Constants::PROJECT_DIR,
task: 'assembleDebugAndroidTest')

  # RUN tests on AWS Device Farm
  aws_device_farm(
    binary_path:         'XYZ/XYZ/build/outputs/apk/XYZ/debug/XYZ-XYZ-debug.apk',
    test_binary_path:    'XYZ/XYZ/build/outputs/apk/androidTest/XYZ/debug/XYZ-XYZ-debug-androidTest.apk'
  )
end

  def createAndroidBuildType(buildType)
    return buildType == "store" ? "Release" : "Alpha"
  end

  def createBuildNumberFromBranch()
    buildNumber = sh "git show -s -n1 --format=%ci"
    return Time.parse(buildNumber).strftime("%Y%m%d-%H%M%S")
  end

  def cleanBranchName(branch)
    return branch
        .gsub('/', "_")
        .gsub('.', "_")
  end

  after_all do |lane|
    # This block is called, only if the executed lane was successful
  end
end

# More information about multiple platforms in fastlane: https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Platforms.md
# All available actions: https://docs.fastlane.tools/actions

# fastlane reports which actions are used
# No personal data is sent or shared. Learn more at https://github.com/fastlane/enhancer

```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby
json_key_file "" # Path to the json secret file - Follow https://github.com/fastlane/supply#setup to get one
package_name "com.XYZ" # e.g. com.krausefx.app

for_lane 'other' do
  package_name "com.XYZ.alpha"
end

for_lane 'beta' do
  package_name "com.XYZ.alpha"
end

for_lane 'store' do
  package_name "com.XYZ"
end

```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.101.1 | ✅ Up-To-Date  |


### Loaded fastlane plugins:

| Plugin                          | Version | Update-Status |
| ------------------------------- | ------- | ------------- |
| fastlane-plugin-aws_device_farm | 0.3.6   | ✅ Up-To-Date  |


<details><summary><b>Loaded gems</b></summary>

| Gem                                     | Version      |
| --------------------------------------- | ------------ |
| did_you_mean                            | 1.0.0        |
| slack-notifier                          | 2.3.2        |
| atomos                                  | 0.1.3        |
| claide                                  | 1.0.2        |
| colored2                                | 3.1.2        |
| nanaimo                                 | 0.2.6        |
| xcodeproj                               | 1.5.9        |
| rouge                                   | 2.0.7        |
| xcpretty                                | 0.2.8        |
| terminal-notifier                       | 1.8.0        |
| unicode-display_width                   | 1.4.0        |
| terminal-table                          | 1.8.0        |
| plist                                   | 3.4.0        |
| addressable                             | 2.5.2        |
| multipart-post                          | 2.0.0        |
| word_wrap                               | 1.0.0        |
| public_suffix                           | 2.0.5        |
| tty-screen                              | 0.6.5        |
| tty-cursor                              | 0.6.0        |
| tty-spinner                             | 0.8.0        |
| babosa                                  | 1.0.2        |
| colored                                 | 1.2          |
| highline                                | 1.7.10       |
| commander-fastlane                      | 4.4.6        |
| excon                                   | 0.62.0       |
| faraday                                 | 0.15.2       |
| unf_ext                                 | 0.0.7.5      |
| unf                                     | 0.1.4        |
| domain_name                             | 0.5.20180417 |
| http-cookie                             | 1.0.3        |
| faraday-cookie_jar                      | 0.0.6        |
| fastimage                               | 2.1.3        |
| gh_inspector                            | 1.1.3        |
| mini_magick                             | 4.5.1        |
| multi_json                              | 1.13.1       |
| multi_xml                               | 0.6.0        |
| rubyzip                                 | 1.2.1        |
| security                                | 0.1.3        |
| xcpretty-travis-formatter               | 1.0.0        |
| dotenv                                  | 2.5.0        |
| bundler                                 | 1.16.3       |
| faraday_middleware                      | 0.12.2       |
| naturally                               | 2.2.0        |
| simctl                                  | 1.6.5        |
| declarative                             | 0.0.10       |
| declarative-option                      | 0.1.0        |
| representable                           | 3.0.4        |
| mime-types-data                         | 3.2016.0521  |
| mime-types                              | 3.1          |
| jwt                                     | 2.1.0        |
| memoist                                 | 0.16.0       |
| os                                      | 1.0.0        |
| signet                                  | 0.8.1        |
| googleauth                              | 0.6.4        |
| httpclient                              | 2.8.3        |
| google-api-client                       | 0.23.4       |
| emoji_regex                             | 0.1.1        |
| json                                    | 2.1.0        |
| io-console                              | 0.4.5        |
| CFPropertyList                          | 3.0.0        |
| libxml-ruby                             | 2.9.0        |
| jmespath                                | 1.4.0        |
| aws-partitions                          | 1.97.0       |
| aws-sigv4                               | 1.0.3        |
| aws-eventstream                         | 1.0.1        |
| aws-sdk-core                            | 3.24.0       |
| aws-sdk-acm                             | 1.9.0        |
| aws-sdk-acmpca                          | 1.2.0        |
| aws-sdk-apigateway                      | 1.13.0       |
| aws-sdk-alexaforbusiness                | 1.7.0        |
| aws-sdk-appstream                       | 1.13.0       |
| aws-sdk-appsync                         | 1.4.0        |
| aws-sdk-applicationautoscaling          | 1.11.0       |
| aws-sdk-applicationdiscoveryservice     | 1.2.0        |
| aws-sdk-athena                          | 1.1.0        |
| aws-sdk-autoscaling                     | 1.6.0        |
| aws-sdk-autoscalingplans                | 1.3.0        |
| aws-sdk-batch                           | 1.5.0        |
| aws-sdk-budgets                         | 1.8.0        |
| aws-sdk-cloud9                          | 1.2.0        |
| aws-sdk-clouddirectory                  | 1.5.0        |
| aws-sdk-cloudformation                  | 1.6.0        |
| aws-sdk-cloudfront                      | 1.4.0        |
| aws-sdk-cloudhsm                        | 1.4.0        |
| aws-sdk-cloudhsmv2                      | 1.3.0        |
| aws-sdk-cloudsearch                     | 1.1.0        |
| aws-sdk-cloudsearchdomain               | 1.1.0        |
| aws-sdk-cloudtrail                      | 1.1.0        |
| aws-sdk-cloudwatch                      | 1.6.0        |
| aws-sdk-cloudwatchevents                | 1.4.0        |
| aws-sdk-cloudwatchlogs                  | 1.4.0        |
| aws-sdk-codebuild                       | 1.13.0       |
| aws-sdk-codecommit                      | 1.4.0        |
| aws-sdk-codedeploy                      | 1.6.0        |
| aws-sdk-codepipeline                    | 1.5.0        |
| aws-sdk-codestar                        | 1.2.0        |
| aws-sdk-cognitoidentity                 | 1.1.0        |
| aws-sdk-cognitoidentityprovider         | 1.5.0        |
| aws-sdk-cognitosync                     | 1.1.0        |
| aws-sdk-comprehend                      | 1.3.0        |
| aws-sdk-configservice                   | 1.13.0       |
| aws-sdk-connect                         | 1.2.0        |
| aws-sdk-costexplorer                    | 1.7.0        |
| aws-sdk-costandusagereportservice       | 1.1.0        |
| aws-sdk-dax                             | 1.1.0        |
| aws-sdk-dlm                             | 1.0.0        |
| aws-sdk-datapipeline                    | 1.1.0        |
| aws-sdk-databasemigrationservice        | 1.7.0        |
| aws-sdk-devicefarm                      | 1.7.0        |
| aws-sdk-directconnect                   | 1.3.0        |
| aws-sdk-directoryservice                | 1.3.0        |
| aws-sdk-dynamodb                        | 1.10.0       |
| aws-sdk-dynamodbstreams                 | 1.1.0        |
| aws-sdk-ec2                             | 1.40.0       |
| aws-sdk-ecr                             | 1.4.0        |
| aws-sdk-ecs                             | 1.16.0       |
| aws-sdk-efs                             | 1.2.0        |
| aws-sdk-eks                             | 1.1.0        |
| aws-sdk-emr                             | 1.3.0        |
| aws-sdk-elasticache                     | 1.4.0        |
| aws-sdk-elasticbeanstalk                | 1.8.0        |
| aws-sdk-elasticloadbalancing            | 1.3.0        |
| aws-sdk-elasticloadbalancingv2          | 1.12.0       |
| aws-sdk-elastictranscoder               | 1.1.0        |
| aws-sdk-elasticsearchservice            | 1.7.0        |
| aws-sdk-fms                             | 1.1.0        |
| aws-sdk-firehose                        | 1.4.0        |
| aws-sdk-gamelift                        | 1.5.0        |
| aws-sdk-glacier                         | 1.9.0        |
| aws-sdk-glue                            | 1.11.0       |
| aws-sdk-greengrass                      | 1.5.0        |
| aws-sdk-guardduty                       | 1.5.0        |
| aws-sdk-health                          | 1.2.0        |
| aws-sdk-iam                             | 1.6.0        |
| aws-sigv2                               | 1.0.1        |
| aws-sdk-importexport                    | 1.1.0        |
| aws-sdk-inspector                       | 1.6.0        |
| aws-sdk-iot                             | 1.11.0       |
| aws-sdk-iot1clickdevicesservice         | 1.1.0        |
| aws-sdk-iot1clickprojects               | 1.1.0        |
| aws-sdk-iotanalytics                    | 1.3.0        |
| aws-sdk-iotdataplane                    | 1.1.0        |
| aws-sdk-iotjobsdataplane                | 1.1.0        |
| aws-sdk-kms                             | 1.7.0        |
| aws-sdk-kinesis                         | 1.4.0        |
| aws-sdk-kinesisanalytics                | 1.3.0        |
| aws-sdk-kinesisvideo                    | 1.2.0        |
| aws-sdk-kinesisvideoarchivedmedia       | 1.2.0        |
| aws-sdk-kinesisvideomedia               | 1.1.0        |
| aws-sdk-lambda                          | 1.8.0        |
| aws-sdk-lambdapreview                   | 1.1.0        |
| aws-sdk-lex                             | 1.4.0        |
| aws-sdk-lexmodelbuildingservice         | 1.6.0        |
| aws-sdk-lightsail                       | 1.5.0        |
| aws-sdk-mq                              | 1.2.0        |
| aws-sdk-mturk                           | 1.4.0        |
| aws-sdk-machinelearning                 | 1.1.0        |
| aws-sdk-macie                           | 1.1.0        |
| aws-sdk-marketplacecommerceanalytics    | 1.1.0        |
| aws-sdk-marketplaceentitlementservice   | 1.1.0        |
| aws-sdk-marketplacemetering             | 1.1.0        |
| aws-sdk-mediaconvert                    | 1.6.0        |
| aws-sdk-medialive                       | 1.8.0        |
| aws-sdk-mediapackage                    | 1.3.0        |
| aws-sdk-mediastore                      | 1.2.0        |
| aws-sdk-mediastoredata                  | 1.2.0        |
| aws-sdk-mediatailor                     | 1.2.0        |
| aws-sdk-migrationhub                    | 1.3.0        |
| aws-sdk-mobile                          | 1.1.0        |
| aws-sdk-neptune                         | 1.2.0        |
| aws-sdk-opsworks                        | 1.4.0        |
| aws-sdk-opsworkscm                      | 1.3.0        |
| aws-sdk-organizations                   | 1.10.0       |
| aws-sdk-pi                              | 1.1.0        |
| aws-sdk-pinpoint                        | 1.6.0        |
| aws-sdk-polly                           | 1.8.0        |
| aws-sdk-pricing                         | 1.1.0        |
| aws-sdk-rds                             | 1.23.0       |
| aws-sdk-redshift                        | 1.6.0        |
| aws-sdk-rekognition                     | 1.5.0        |
| aws-sdk-resourcegroups                  | 1.2.0        |
| aws-sdk-resourcegroupstaggingapi        | 1.1.0        |
| aws-sdk-route53                         | 1.10.1       |
| aws-sdk-route53domains                  | 1.3.0        |
| aws-sdk-s3                              | 1.17.0       |
| aws-sdk-ses                             | 1.8.0        |
| aws-sdk-sms                             | 1.1.0        |
| aws-sdk-sns                             | 1.3.0        |
| aws-sdk-sqs                             | 1.4.0        |
| aws-sdk-ssm                             | 1.20.0       |
| aws-sdk-swf                             | 1.1.0        |
| aws-sdk-sagemaker                       | 1.13.0       |
| aws-sdk-sagemakerruntime                | 1.1.0        |
| aws-sdk-secretsmanager                  | 1.10.0       |
| aws-sdk-serverlessapplicationrepository | 1.4.0        |
| aws-sdk-servicecatalog                  | 1.6.0        |
| aws-sdk-servicediscovery                | 1.3.0        |
| aws-sdk-shield                          | 1.3.0        |
| aws-sdk-simpledb                        | 1.1.0        |
| aws-sdk-snowball                        | 1.4.0        |
| aws-sdk-states                          | 1.3.0        |
| aws-sdk-storagegateway                  | 1.7.0        |
| aws-sdk-support                         | 1.1.0        |
| aws-sdk-transcribeservice               | 1.4.0        |
| aws-sdk-translate                       | 1.2.0        |
| aws-sdk-waf                             | 1.5.0        |
| aws-sdk-wafregional                     | 1.5.0        |
| aws-sdk-workdocs                        | 1.2.0        |
| aws-sdk-workmail                        | 1.2.0        |
| aws-sdk-workspaces                      | 1.3.0        |
| aws-sdk-xray                            | 1.3.0        |
| aws-sdk-resources                       | 3.22.0       |
| aws-sdk                                 | 3.0.1        |
| fastlane-plugin-aws_device_farm         | 0.3.6        |
</details>


*generated on:* **2018-08-07**
</details></pre>
</details>

**Comment by janpio** [comment]
Could you please start a new issue and include all the requested information from the issue template? It will be easier to help you there instead of in this feature request issue by someone else. Thank you.
